### DISCUSSION TITLE
Multi camera http stream

### DISCUSSION THREAD

**Discussion by simondsmason** [blakeblackshear/frigate#4661] [General]
I am moving off Blue Iris and the one feature I still use from BI is the multi-camera stream that I send to my touchscreens.  It has two options - one that creates a mosaic of the cameras - I don't use this one and one that switches between selected cameras with a short delay in-between - I do use this.  Is there anyway to do either of these in frigate currently?  Thanks.

**Comment by NickM-27** [comment]
Frigate has the birdseye view which is a mosaic but can be configured to only show cameras with motion, with objects, or continuously (this is a per camera option)

https://github.com/dermotduffy/frigate-hass-card offers both of these and many more options as well

**Reply by simondsmason** [reply]
Thanks. I am using Birds Eye and I have checked out the Lovelace. Although I am starting to play with HASS it is not my core system. I am using the http streams through frigate to display on sharptools touchscreens. I haven’t figured out a way to get Lovelace to generate http streams with multi camera etc. but maybe I need to look more closely?

**Reply by simondsmason** [reply]
PS. Is there a way to see birdseye in an http stream?  That would be a good start.

**Reply by NickM-27** [reply]
Yes, birdseye can be accessed like any other camera using `http://frigate_ip:5000/api/live/birdseye`

**Reply by simondsmason** [reply]
Sorry - should have been clearer.  I don't want the menu - just the video feed.  I am displaying this in touchscreens.  Currently BI gives me one feed per camera and one feed that switches between the cameras.  The Birdseye http link takes me to the frigate menu.

Here is what it looks like on my touchscreens:

![image](https://user-images.githubusercontent.com/6063262/206861472-505002f3-f38c-479e-8277-adbaa4bd2b42.png)

**Reply by NickM-27** [reply]
> Sorry - should have been clearer. I don't want the menu - just the video feed

that's exactly what I am referring to, a direct video feed using the frigate http api

https://github.com/blakeblackshear/frigate/blob/739a2674621a224c6b1f8323c7430dc2d0c16954/web/src/routes/Birdseye.jsx#L10

it would require some handling to embed though as it is a websocket, 

`http://frigate_ip:5000/api/live/birdseye`

**Reply by simondsmason** [reply]
Thanks, it looks like I need to modify the file Birdseye.jsx.  I am installed as a docker in Unraid.  I looked for the file but couldn't find it?  Thanks.

**Reply by NickM-27** [reply]
There are no modifications needed, you can access the stream itself using `http://frigate_ip:5000/api/live/birdseye` it just won't work in a web browser directly, you'll need a component to work with it (I am not familiar with what software you're using, so can't give specific advice)

but just like the frigate card can show the birdseye stream, any other software that has a jsmpeg player should be able to play it as well

**Reply by simondsmason** [reply]
Got it - thanks - will play around with this.
